src/scenes/Event/components/Sidebar.js

```diff
@@ -34,7 +35,13 @@ const Sidebar = inject('store')(observer(({ store: { eventPage: { oracle } } })
   </SidebarContainer>
 )));
 
-const SidebarContainer = styled(Grid).attrs({ item: true, xs: 12, md: 4 })`
+const SidebarContainer = withStyles(styles)(({ children, classes, ...props }) => (
```

Comment:
no need to pass `...props`. the reason for that was to access the `theme` prop but since you are using JSS it is not necessary.